torchgeo/datamodules/eurosat.py

```diff
@@ -9,7 +9,7 @@
 import pytorch_lightning as pl
 import torch
 from torch.utils.data import DataLoader
-from torchvision.transforms import Compose, Normalize
```

Comment:
Removed Normalize as it is more explicit to subtract band means then divide by band stdevs.

Reply:
Not sure if I agree with this. I think the `Normalize` transform is more explicit since it tells you why you're doing it, not just what you're doing.

Reply:
True, I was thinking of it as, "there are many ways to 'normalize' something where `Normalize` happens to apply channel wise standardization". I don't have a strong opinion here.

Reply:
I don't have a strong preference here either. I tend to use builtin library functions where possible since I trust them more than I trust my own coding, but this one is obviously very simple. I think the biggest advantage of using `Normalize` is that it does the `.reshape(-1, 1, 1)` here for you, so that's one less thing to worry about. It also uses `.view(-1, 1, 1)` which I think is more efficient. It also accepts a list, so you don't need to convert it to a tensor yourself. Personally I'm gonna vote for using `Normalize` to keep the code as simple as possible.